cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -92,6 +92,8 @@ private RouteDestination getV3RoutingService(String [] uriParts, AllowedMethod r
     if ((uriParts.length >= 2) && uriParts[1].equals("feeds")) {
       // TODO find a better way to handle that - this looks hackish
       return null;
+    } else if ("dashboard".equals(uriParts[1])) {
```

Comment:
if you make changes here, please add a test case in `RouterPathTest` with sample endpoints.